src/components/Contact/ForCompanies.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import styled from "styled-components";
+import devices from '../../utility/MediaQueries.js';
+
+const SVGContainer = styled.div`
+    min-width: 50vw;
+    ${devices.tablet`
+        order: 3;
+        width: 100vw;
+    `}
+`;
+
+
+const ForCompanies = () =>
+    <React.Fragment>
+        <SVGContainer>
+        <svg viewBox="0 0 688 550" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You'll want to save this into a file instead and then import it. See #102 for how I do it except that you'll either use it as the source for an `img`. If you have issues let me know.